hal/src/rtl872x/sleep_hal.cpp

```diff
@@ -428,21 +457,61 @@ class SleepClass {
             PSRAM_datawr = ((temp << 8) | (temp << 24)) | (temps | (temps << 16));
             PSRAM_CTRL_CA_Gen(psram_ca, (BIT11 | BIT0), PSRAM_LINEAR_TYPE, PSRAM_REG_SPACE, PSRAM_WRITE_TRANSACTION);
             PSRAM_CTRL_DPin_Reg(psram_ca, &PSRAM_datawr, PSRAM_WRITE_TRANSACTION);
+
+            // Disable memory access from CPU
```

Comment:
Was this supposed to be commented our or not? 
We shouldnt need to disable CPU access to PSRAM since we are going to sleep here shortly anyways (scheduler is disabled, IRQs disabled, nothing else should execute that could access PSRAM)?

Reply:
Ideally I'd love to have it here, but this causes problems on boot-up if reset while in sleep, so better to leave commented out.